README.md

```diff
@@ -32,6 +33,22 @@ The above command will work on just about any system, but you can follow the bel
                 1. Source Type / Source: Provide approrpriate values that will allow your database clients to connect to this port
     1. Launch Instance!
 1. SSH to the new instance and install the sidecar with the above command
+</details>
+
+<details>
+    <summary>Azure VM</summary>
+
+1. Go to [Virtaual Machines](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Compute%2FVirtualMachines)
+1. Select Create -> [Azure virtual Machine](https://portal.azure.com/#create/Microsoft.VirtualMachine)
+1. Required fields outlined below
+    1. Image: Ubuntu Server 20.04 is the optimal option, however other images should work well too
```

Comment:
Should we change this to `other linux based images` and the same with the above instructions to be safe?

Reply:
agreed